### PR TITLE
Create methods "translateAllBreedsByLinguatools", "createBreedTransla…

### PR DESCRIPTION
**Pull request by udodikdmitro**
…tion"

### CODE REVIEW

File: src/test/java/com/catapi/service/TranslationServiceTest.java [hunk 1]
```diff
@@ -198,4 +204,40 @@ private String getTranslationResponse(String testToTranslate) {
...
+
+        translationService.translateAllBreedsByLinguatools(Locale.UK);
+
+        ArgumentCaptor<BreedTranslation> breedCaptor = ArgumentCaptor.forClass(BreedTranslation.class);
+        verify(breedTranslationRepository, times(1)).save(breedCaptor.capture());
+
+        BreedTranslation saveBreedTranslation = breedCaptor.getAllValues().getFirst();
+        assertEquals("Порода2", saveBreedTranslation.getBreedName());
+        assertEquals("Опис2", saveBreedTranslation.getDescription());
+    }
```
Comment (HailToSG) [inline src/test/java/com/catapi/service/TranslationServiceTest.java]: private methods should go after all the rest

File: src/main/java/com/catapi/service/TranslationService.java [hunk 2]
```diff
@@ -121,4 +130,39 @@ private String extractTranslation(String scriptContent) {
         }
         throw new ExternalApiException("Wrong response format");
     }
+
+    public String translateAllBreedsByLinguatools(Locale locale) {
```
Comment (HailToSG) [inline src/main/java/com/catapi/service/TranslationService.java]: You DON'T need to return String here

**Review by HailToSG — CHANGES_REQUESTED** [review]

File: src/main/java/com/catapi/controller/TranslationController.java [hunk 3]
```diff
@@ -38,4 +42,22 @@ public void updateTranslationManually(@PathVariable Long catFactTranslationId, @
...
+        try {
+            if (EnumSet.of(Locale.UK, Locale.RU).contains(Locale.valueOf(locale.toString()))) {
+                return ResponseEntity.ok(translationService.translateAllBreedsByLinguatools(locale));
+            } else {
+                throw new IllegalArgumentException(STR."Can not get translation with using parameter \{locale}");
+            }
+        } catch (IllegalArgumentException e) {
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(e.getMessage());
+        }
+    }
```
Comment (HailToSG) [inline src/main/java/com/catapi/controller/TranslationController.java]: You don't need to

1. return ResponseEntity<String>
2. check locale
3. put logic in controllers

**Review by HailToSG — CHANGES_REQUESTED** [review]

File: src/test/java/com/catapi/service/TranslationServiceTest.java (hunk 1 above)
Comment (udodikdmitro) [reply src/test/java/com/catapi/service/TranslationServiceTest.java]: Moved

File: src/main/java/com/catapi/service/TranslationService.java (hunk 2 above)
Comment (udodikdmitro) [reply src/main/java/com/catapi/service/TranslationService.java]: Fixed

File: src/main/java/com/catapi/controller/TranslationController.java (hunk 3 above)
Comment (udodikdmitro) [reply src/main/java/com/catapi/controller/TranslationController.java]: Fixed

**Review by HailToSG — APPROVED** [review]

**Pull request merged**